src/test/java/org/opensearch/security/dlic/rest/api/UserApiTest.java

```diff
@@ -86,7 +87,7 @@ public void testSecurityRoles() throws Exception {
                 .executeGetRequest(ENDPOINT + "/" + CType.INTERNALUSERS.toLCString());
         Assert.assertEquals(response.getBody(), HttpStatus.SC_OK, response.getStatusCode());
         Settings settings = Settings.builder().loadFromSource(response.getBody(), XContentType.JSON).build();
-        Assert.assertEquals(133, settings.size());
+        Assert.assertEquals(171, settings.size());
```

Comment:
I really hate this, consider cleaning this up in a follow up PR.  As it stands these tests just config an arbitrary size instead of the contents.

If there is a math based source to this value make a constant and preform a calcuation that can be followed by a human such as `final int USER_SETTING_SIZE = 9 /* lines per entry*/ * 19 /* number of accounts*/;` so that if 19 -> 20 its should be obvious as part of the code changes why this was done